README.rst

```diff
@@ -79,12 +79,17 @@ You need a VIM version that was compiled with Python 2.7 or later
 
 Manual installation
 -------------------
+The plugin uses `git` submodules to fetch latest version of Jedi.
```

Comment:
```suggestion
The plugin uses `git` submodules to fetch the vendored versions of Jedi.
```
(I've figured it would be better to not include details about "and its dependencies (parso)")